parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -102,6 +102,7 @@ updateAptWithMicrosoftPkg() {
     fi
 
     retrycmd_if_failure 10 5 10 cp /tmp/microsoft-prod.list /etc/apt/sources.list.d/ || exit $ERR_MOBY_APT_LIST_TIMEOUT
+    echo "deb [arch=amd64,arm64,armhf] https://packages.microsoft.com/ubuntu/18.04/multiarch/prod testing main" > /etc/apt/sources.list.d/microsoft-prod-testing.list
```

Comment:
please use UBUNTU_RELEASE here for 20.04, 22.04

Reply:
nvm, I see it doesn't exist for 20.04 and 22.04...something to perhaps follow up on

Reply:
Those repos don't have "multiarch" in the url.

Reply:
@cpuguy83, thanks for the help with this! Probably a basic question: do the containerd builds for 20.04 and 22.04 differ, from the 18.04 builds? Wondering if it's ok to use the 18.04 URL for all (or change it depending on the version).

Reply:
@ganeshkumar5699 Yes, the builds are different as the binaries are dynamically linked against the toolchain available for each distro.

Reply:
Thank you. Will likely be making this change in a subsequent PR.